src/App.js

```diff
@@ -1,15 +1,62 @@
-import React from "react";
-import Button from "./components/button";
+import React, { useEffect, useState } from "react";
+import Recipe from "./components/Recipe";
+
+const App = () => {
+  const APP_ID = "e74fe7a6";
+  const API_KEY = "8c5d31e7816e4c8b54e498f9ac5b7377";
+
```

Comment:
just keeping here for now before pulling into separate constants file

Reply:
Defining them inside here means they will be recreated on every render. Will make pretty much no difference but it's a little cleaner to define them outside the component